embulk-output-redshift/src/main/java/org/embulk/output/redshift/RedshiftOutputConnection.java

```diff
@@ -28,8 +28,6 @@ public RedshiftOutputConnection(Connection connection, String schemaName, boolea
         connection.setAutoCommit(autoCommit);
     }
 
-    // Redshift does not support DROP TABLE IF EXISTS.
-    // Here runs DROP TABLE and ignores errors.
     @Override
     public void dropTableIfExists(TableIdentifier table) throws SQLException
```

Comment:
We can remove this method because the super class (AbstractJdbcOutputPlugin) does same thing.